subworkflows/local/long_reads_assembly_coverage.nf

```diff
@@ -0,0 +1,46 @@
+include { MINIMAP2_ALIGN as MINIMAP_COVERAGE   } from '../../modules/nf-core/minimap2/align/main'
+include { SAMTOOLS_IDXSTATS                    } from '../../modules/nf-core/samtools/idxstats/main'
+include { METABAT2_JGISUMMARIZEBAMCONTIGDEPTHS } from '../../modules/nf-core/metabat2/jgisummarizebamcontigdepths/main'
+
+workflow LONG_READS_ASSEMBLY_COVERAGE {
+
+    take:
+    assembly_reads   // [ val(meta), path(assembly_fasta), path(reads) ]
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    reads = assembly_reads.map { meta, _, reads -> [meta, reads] }
+    assembly = assembly_reads.map { meta, assembly, _ -> [meta, assembly] }
+
+    MINIMAP_COVERAGE(
```

Comment:
I guess we are storing bam-files here.. maybe should consider similar approach like in GGP in order to save storage @mberacochea

Reply:
ahh yep, I missed that one. BAM files are _evil_, _pure evil_.